packages/@aws-cdk/core/lib/context-provider.ts

```diff
@@ -81,7 +82,7 @@ export class ContextProvider {
     };
   }
 
-  public static getValue(scope: Construct, options: GetContextValueOptions): any {
+  public static getValue(scope: Construct, options: GetContextValueOptions): GetContextValueResult {
```

Comment:
I am wondering why did we need to change the return type here. Seems like it doesn't provide any benefit.

Reply:
Consistency with `getKey()`.